src/test/java/org/dinosaur/foodbowl/domain/member/persistence/MemberRepositoryTest.java

```diff
@@ -18,27 +18,22 @@ class MemberRepositoryTest extends PersistenceTest {
     private MemberRepository memberRepository;
 
     @Test
-    void 회원_저장() {
-        Member member = Member.builder()
-                .email("foodBowl@gmail.com")
-                .socialId("foodBowlId")
-                .socialType(SocialType.APPLE)
-                .nickname("foodbowl")
-                .introduction("안녕하세요")
-                .build();
+    void 아이디와_일치하는_회원_조회() {
+        Member saveMember = memberTestPersister.memberBuilder().save();
 
-        Member saveMember = memberRepository.save(member);
+        Member findMember = memberRepository.findById(saveMember.getId()).get();
 
-        assertThat(saveMember.getId()).isNotNull();
+        assertThat(findMember.getId()).isEqualTo(saveMember.getId());
```

Comment:
아래의 썸네일과 함께 조회 테스트와 동일하게 검증할 수 있을 것 같아요 !

```suggestion
        assertThat(findMember).isEqualTo(saveMember);
```